bot.py

```diff
@@ -15,14 +16,33 @@
 
 user = git.get_user()
 
+with open('repos.json') as f:
+  reposDict = json.load(f)
+
+  
+
+@bot.command(brief="associate a repository to this channel")
+async def associate(ctx, repoName): #!git associate repo
+	guild = ctx.message.guild.name
+	channel = ctx.message.channel.id
+	reposDict[channel]=repoName
```

Comment:
I wonder how dangerous this dict being global is, especially if we're dealing with `async` functions. Perhaps making a function `update_reposDict` that [locks](https://stackoverflow.com/a/25799871/6032037) the variable could be a better way to go